### PR TITLE
Stream API, word's frequency in the text

### PR DESCRIPTION
**Pull request by nvkaip**

### CODE REVIEW

File: hw14/WordsCountApp.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                    }
+                    if (collectionOfWords.get(e1) < (collectionOfWords.get(e2))){
+                        return 1;
+                    }else {
+                        return e1.compareTo(e2);
+                    }
+                })
+                .limit(10)
+                .forEach(System.out::println);
+        scanner.close();
```
Comment (Sofasmile) [inline hw14/WordsCountApp.java]: do we need this line here?

File: hw14/WordsCountApp.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import java.util.stream.Stream;
+
+public class WordsCountApp {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String stringOfWords = scanner.nextLine().toLowerCase();
+        Map<String, Long> collectionOfWords = Stream.of(stringOfWords
+                .split("[^a-zA-Zа-яА-Я0-9]+"))
+                .collect(Collectors.groupingBy(word -> word, Collectors.counting()));
+        collectionOfWords.keySet().stream()
```
Comment (Sofasmile) [inline hw14/WordsCountApp.java]: better:
```suggestion
        collectionOfWords.keySet()
                .stream()
```

File: hw14/WordsCountApp.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                .sorted((e1, e2) -> {
+                    if ((collectionOfWords.get(e1) - collectionOfWords.get(e2)) != 0) {
+                        return (int) (collectionOfWords.get(e2) - collectionOfWords.get(e1));
+                    } else {
+                        return e1.compareTo(e2);
+                    }
+                })
+                .limit(10)
+                .forEach(System.out::println);
+        scanner.close();
```
Comment (Sofasmile) [inline hw14/WordsCountApp.java]: do we need this line here?

File: hw14/WordsCountApp.java (hunk 1 above)
Comment (nvkaip) [reply hw14/WordsCountApp.java]: well, it's not in the try, so I thought it will not close... will it?

**Pull request merged**